src/main/java/org/veupathdb/eda/binaryfiles/printer/BinaryFilePrinter.java

```diff
@@ -86,15 +78,19 @@ private static void printAncestorFile(Path binaryFile, JSONObject metajson) {
         }
 
       } catch (IOException e) {
-      throw new RuntimeException("Failed attempting to read file " + binaryFile.toString(), e);
+      throw new RuntimeException("Failed attempting to read file " + binaryFile, e);
     }
   }    
   
   private static void printIdsMapFile(Path binaryFile, JSONObject metajson) {
+    List<Integer> bytesReservedPerAncestors = new ArrayList<>();
+    int bytesReservedForId = metajson.getInt("bytesReservedForId");
+    JSONArray ancestorBytesReserved = metajson.getJSONArray("bytesReservedPerAncestor");
```

Comment:
These property names should probably be constants.  Missed where they are set but should be used there too.

Reply:
Thanks, I extracted them to constants and missed updating here.